tests/Makefile

```diff
@@ -287,11 +287,15 @@ ifneq (,$(filter MINGW% MSYS%,$(UNAME)))
   HOST_OS = MSYS
 endif
 
+ifneq (,$(filter CYGWIN%,$(UNAME)))
+  HOST_OS = CYGWIN
```

Comment:
minor: isn't cygwin a `posix` environment ?

Reply:
Kinda, but in this scenario it should be.